docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
     container_name: teacher-vacancy-service_web_1
     environment:
       RAILS_ENV: "development"
+      DATABASE_URL: "postgres://postgres@db:5432/tvs_development?template=template0&pool=5&encoding=unicode"
```

Comment:
A neat way to address the problem, I'm in favour of setting these variables here rather than docker-compose.env with an attempt to overwrite only in test, much cleaner and easier to follow what's going on 👍 

I've been playing around with it to test in my environment and still get the same error. I've spent some further time trying to poke it to see if I can fix it but no joy.

```
ActiveRecord::EnvironmentMismatchError: You are attempting to modify a database that was last run in `test` environment.
You are running in `development` environment. If you are sure you want to continue, first set the environment using:

        bin/rails db:environment:set RAILS_ENV=development
```

Requires a bit more digging I think and happy to screen share to help debug it.

Reply:
Just to check: did you remove `DATABASE_URL` (and `ELASTICSEARCH_URL`) from your `docker-compose.env`?

Reply:
Weird it works now, they were both omitted before though!

Reply:
Odd. Glad it's working now, though.